desktop/components/my_active_bids/index.styl

```diff
@@ -37,6 +37,10 @@ item-margin-bottom = 16px
 .my-active-bids-item-details
   line-height 20px
 
+.my-active-bids-item-details--hide-bid
+  @extends .my-active-bids-item-details
+  padding-top 10px
+
```

Comment:
I was wondering about this- the `--hide-bid` classname was a little confusing to me since it doesn't seem to be hiding anything. Super minor and I was able to put everything together with a minute of looking at the template.

I think the different component types could be organized in a more bem-y way to make this and the template really explicit between class and layout but I am not sure the trade off would be worth it.